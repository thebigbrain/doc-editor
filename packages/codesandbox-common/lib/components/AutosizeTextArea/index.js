"use strict";

var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;

var _reactTextareaAutosize = _interopRequireDefault(require("react-textarea-autosize"));

var _Input = _interopRequireDefault(require("../Input"));

var _default = _Input["default"].withComponent(_reactTextareaAutosize["default"]);

exports["default"] = _default;