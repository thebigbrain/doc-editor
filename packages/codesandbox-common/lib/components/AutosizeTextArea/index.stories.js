"use strict";

var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");

var _react = _interopRequireDefault(require("react"));

var _react2 = require("@storybook/react");

var _ = _interopRequireDefault(require("."));

var stories = (0, _react2.storiesOf)('components/Input', module);
stories.add('Basic AutosizeTextArea', function () {
  return _react["default"].createElement(_["default"], {
    value: "I am a fancy textarea"
  });
});