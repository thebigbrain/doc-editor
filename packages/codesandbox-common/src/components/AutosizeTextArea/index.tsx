import AutosizeInput from 'react-textarea-autosize';
import Input from '../Input';

export default Input.withComponent(AutosizeInput);
