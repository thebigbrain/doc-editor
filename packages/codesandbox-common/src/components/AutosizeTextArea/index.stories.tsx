import React from 'react';
import { storiesOf } from '@storybook/react';
import AutosizeTextArea from '.';

const stories = storiesOf('components/Input', module);
stories.add('Basic AutosizeTextArea', () => (
  <AutosizeTextArea value="I am a fancy textarea" />
));
