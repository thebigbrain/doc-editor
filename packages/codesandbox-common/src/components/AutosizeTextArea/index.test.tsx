import 'jest-styled-components';
import React from 'react';
import mountWithTheme from '../../test/themeMount';
import AutosizeTextArea from '.';

describe('<AutosizeTextArea /> rendering', () => {
  it('renders correctly', () => {
    expect(mountWithTheme(<AutosizeTextArea />)).toMatchSnapshot();
  });
});
