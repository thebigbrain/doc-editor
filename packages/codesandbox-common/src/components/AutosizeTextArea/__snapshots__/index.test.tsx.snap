// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AutosizeTextArea /> rendering renders correctly 1`] = `ReactWrapper {}`;
